tests/exec_name/test

```diff
@@ -51,48 +58,62 @@ ok( $exec  ne "" and $exec2 ne "" );
 );
 
 sub do_test {
-    my $rule = shift(@_);
+    my ( $rules, $filters, $f1, $f2, $f3 ) = @_;
```

Comment:
I think we can do a better naming job with $rules and $filters.  Especially since they could both be considered "rules" and/or "filters".

How about "filter_always" and "filter_exclude"?

Reply:
Good point, I renamed them to `$exit` and `$exclude`.